dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -150,6 +151,54 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         }
     }
 
+    void AppNotificationManager::Register(hstring const& displayName, winrt::Windows::Foundation::Uri const& iconUri)
+    {
+        if (!IsSupported())
+        {
+            return;
+        }
+
+        HRESULT hr{ S_OK };
```

Comment:
NIT: `HRESULT hr{};`

No need to specify the defaults if they're 0/0.0/false/nullptr (they're the defaults)